pkg/cnb/builder_builder.go

```diff
@@ -172,7 +171,21 @@ func (bb *BuilderBuilder) writeableImage() (v1.Image, error) {
 	})
 }
 
-func (bb *BuilderBuilder) lifecycleLayer() (v1.Layer, error) {
+func (bb *builderBlder) validateBuilder() {
+	for _, bp := range bb.buildpackLayers {
+		err := bp.BuildpackLayerInfo.supports(bb.stackId)
+		if err != nil {
+			bb.err = errors.Wrapf(err, "validating buildpack: %s", bp.BuildpackInfo)
+			return
```

Comment:
i wonder if we should go through every buildpack in the spec and list all the once that are unsupported

Reply:
That would require us to combine the errors. Which would be great :)